DnDCharacterSheet/Services/SheetService.cs

```diff
@@ -8,15 +9,15 @@ public class SheetService : ISheetService
     {
         private IAttributeSettingStrategy? _attributeSettingStrategy;
 
-        public void SetStrategy(IAttributeSettingStrategy strategy)
+        public void SetAttributeSettingStrategy(IAttributeSettingStrategy strategy)
         {
             _attributeSettingStrategy = strategy;
         }
 
-        public Sheet SetStrengthAttribute(Sheet sheet, int value)
+        public Sheet SetAttribute(Sheet sheet, int value)
         {
             return _attributeSettingStrategy != null ?
-                _attributeSettingStrategy.SetStrengthAttribute(sheet, value)
+                _attributeSettingStrategy.SetAttribute(sheet, value)
                 : throw new Exception(Constants.SheetService.NoStrategyError);
         }
```

Comment:
all of these code should not be allowed to work with a null reference, therefore avoid processing any nullable code